src/test/java/mate/academy/security/AuthenticationServiceImplTest.java

```diff
@@ -0,0 +1,105 @@
+package mate.academy.security;
+
+import static org.mockito.ArgumentMatchers.any;
+
+import java.util.Optional;
+import java.util.Set;
+import mate.academy.exception.AuthenticationException;
+import mate.academy.model.Role;
+import mate.academy.model.User;
+import mate.academy.service.RoleService;
+import mate.academy.service.UserService;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import org.springframework.security.crypto.password.PasswordEncoder;
+
+class AuthenticationServiceImplTest {
+    private static final String EMAIL = "bob@i.ua";
+    private static final String PASSWORD = "12345";
+    private static final Role ROLE_USER = new Role(Role.RoleName.USER);
+    private User user;
+    private Optional<User> optionalUser;
+    private UserService userService;
+    private RoleService roleService;
+    private PasswordEncoder passwordEncoder;
```

Comment:
This variable is used only for creating AuthenticationService, so you can move it ti setUp() and make them local
(I think IDE said that it could be local, too)